Workaround auto darkmode issue, image are not inverted correctly in darkmode

diff -up chromium-148.0.7778.96/third_party/blink/renderer/platform/graphics/dark_mode_image_classifier.cc.me chromium-148.0.7778.96/third_party/blink/renderer/platform/graphics/dark_mode_image_classifier.cc
--- chromium-148.0.7778.96/third_party/blink/renderer/platform/graphics/dark_mode_image_classifier.cc.me	2026-05-06 16:22:25.906024153 +0200
+++ chromium-148.0.7778.96/third_party/blink/renderer/platform/graphics/dark_mode_image_classifier.cc	2026-05-06 16:22:52.465423557 +0200
@@ -265,7 +265,7 @@ DarkModeResult DarkModeImageClassifier::
 
   // Very few colors means it's not a photo, apply the filter.
   if (features.color_buckets_ratio < low_color_count_threshold)
-    return DarkModeResult::kApplyFilter;
+    return DarkModeResult::kDoNotApplyFilter;
 
   // Too many colors means it's probably photorealistic, do not apply it.
   if (features.color_buckets_ratio > high_color_count_threshold)
